legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-core/src/main/java/org/finos/legend/engine/persistence/components/relational/sqldom/schemaops/statements/SelectStatement.java

```diff
@@ -128,6 +129,15 @@ public void genSql(StringBuilder builder) throws SqlDomException
             }
         }
 
+        // Add limit clause
```

Comment:
Can we change the comment on line 76 to limit

Reply:
Will add that